practice-two/src/styles/utils/unit.scss

```diff
@@ -0,0 +1,5 @@
+@use 'sass:math';
+
+@function toRem($pxValue) {
```

Comment:
Your function has 2 parameters. Why when you using it you pass only one argument?